src/Microsoft.ComponentDetection.Detectors/rust/RustCliDetector.cs

```diff
@@ -118,13 +125,19 @@ private static (string Name, string Version) ParseNameAndVersion(string nameAndV
         string id,
         DetectedComponent parent,
         Dep depInfo,
+        IReadOnlyDictionary<string, (string Authors, string License)> packages,
```

Comment:
nit: can we name this something like packagesMetadata to be clearer since this doesn't contain data on the packages themselves?

Reply:
Changed to packagesMetadata as suggested